external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpoutMessageId.java

```diff
@@ -21,7 +21,9 @@
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 import org.apache.kafka.common.TopicPartition;
 
-public class KafkaSpoutMessageId {
+import java.io.Serializable;
+
+public class KafkaSpoutMessageId implements Serializable{
     private final transient TopicPartition topicPart;
```

Comment:
@hmcl 
I guess the fields are all Serializable. Is that a reason to make them transient?